pkgs/by-name/kc/kcl-language-server/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "kcl-language-server";
+  version = "0.1.61";
+
+  src = fetchFromGitHub {
+    owner = "KittyCAD";
+    repo = "kcl-lsp";
+    rev = "v${version}";
+    hash = "sha256-VtrR4v0BJWYdoYFDJpWnmVqDhZMlPGm+g9yjxYfcFxQ=";
+  };
+
+  cargoHash = "sha256-87S+8uyACWx6URGJGcFdfPbH6UJnR+7685zFSof27Do=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  meta = with lib; {
+    description = "The kcl language server";
+    homepage = "https://github.com/KittyCAD/kcl-lsp";
+    license = licenses.mit;
+    maintainers = [ ];
```

Comment:
New packages must have maintainers.

Reply:
Thanks @Aleksanaa for the feedback. I'm new to contributing and I'd like some advice on finding a maintainer for this. 
Could you point me to some documentations for this please?

Reply:
Since you are adding the package, you should be the maintainer.